From 90496592a0daa4e6c2c7bdbe32c87990be33a658 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 25 Dec 2019 09:26:59 +0800
Subject: [PATCH 1/4] drm/bridge: it6263: drop function drm_bridge_add
 returning value

Upstream commit 9928688492ae (drm/bridge: change return type
of drm_bridge_add function), the function drm_bridge_add has no
returning value anymore.

So drop the return value of it to avoid compile error.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/gpu/drm/bridge/it6263.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/bridge/it6263.c b/drivers/gpu/drm/bridge/it6263.c
index 8123dc34df54..2bc5f5a79621 100644
--- a/drivers/gpu/drm/bridge/it6263.c
+++ b/drivers/gpu/drm/bridge/it6263.c
@@ -906,11 +906,7 @@ static int it6263_probe(struct i2c_client *client,
 
 	it6263->bridge.funcs = &it6263_bridge_funcs;
 	it6263->bridge.of_node = np;
-	ret = drm_bridge_add(&it6263->bridge);
-	if (ret) {
-		dev_err(dev, "Failed to add drm_bridge\n");
-		return ret;
-	}
+	drm_bridge_add(&it6263->bridge);
 
 	i2c_set_clientdata(client, it6263);
 
-- 
2.17.1

